packages/v4/patternfly-docs/content/accessibility/navigation/navigation.md

```diff
@@ -0,0 +1,89 @@
+---
+
+import { Checkbox, List, ListItem } from '@patternfly/react-core';
+
+
+## Accessibility
+
+To implement an accessible PatternFly **navigation** component:
+- Provide an accessible name for the navigation via the `aria-label` attribute
+- For grouped or expandable nav, add a `title` to each section to distinguish them for users. If `title` is not used, an `aria-label` must be used on the `NavGroup` or `NavExpandable` component.
+
+
+
+## Testing
+
+At a minimum, a navigation component should meet the following criteria:
+
+<List isPlain>
+  <ListItem>
+    <Checkbox id="navigation-a11y-checkbox-1" label={<span>The navigation has an accessible name via the <code class="ws-code">aria-label</code></span>} description="This will differentiate each navigation when announced to users of assistive technologies when navigating through a page or a screen reader's rotor menu." />
+  </ListItem>
+  <ListItem>
+    <Checkbox id="navigation-a11y-checkbox-2" label="Standard keyboard navigation can be used to navigate between focusable elements within the navigation." description={<span><kbd>Tab</kbd> navigates to the next focusable element, and <kbd>Shift</kbd> + <kbd>Tab</kbd> navigates to the previous focusable element. Expandable items can be opened with <kbd>Enter</kbd> or <kbd>Space</kbd>.  When the navigation is collapsed, you should not be able to navigate to any of the elements.</span>} />
+  </ListItem>
+  <ListItem>
+    <Checkbox id="navigation-a11y-checkbox-3" label={<span>For expandable or grouped navigation, an accessible name via the <code class="ws-code">title</code> or <code class="ws-code">aria-label</code> is used.</span>} description="This will differentiate each navigation when announced to users of assistive technologies when navigating through a page or a screen reader's rotor menu." />
+  </ListItem>
+</List>
+
+
+
+## React customization
+
+Various React props have been provided for more fine-tuned control over accessibility.
+
+| Prop | Applied to | Reason | 
+|---|---|---|
+| `aria-label` | `Nav` | 	Adds an accessible label for the nav when there are multiple navs on the page |
+| `ariaLeftScroll` | `NavList` | Adds an aria-label for the left scroll button |
+| `ariaRightScroll` | `NavList` | Adds an aria-label for the right scroll button |
+| `id` | `NavGroup` | Used as an identifier to use when there is an `aria-labelledby` on a section element |
+| `title` | `NavGroup` | Adds a title shown for the group |
+| `isActive` | `NavItem` |  Will add the `aria-current` attribute on a nav item when active |
+| `isExpanded` | `NavExpandable` | Adds the `aria-expanded` attribute when is expanded and `hidden` attribute when collapsed |
+| `title` | `NavExpandable` | Adds a title shown for the expandable list |
+| `buttonProps` | `NavExpandable` | Adds any additional props added to the NavExpandable `<button>`|
+| `srText` | `NavExpandable` | If defined, screen readers will read this text instead of the list title |
```

Comment:
```suggestion
| `aria-label="[text describing the nav component]"` | `Nav` | 	Adds an accessible label for the nav when there are multiple navs on the page |
| `ariaLeftScroll="[text describing the left scroll button]"` | `NavList` | Adds an aria-label for the left scroll button |
| `ariaRightScroll="[text describing the right scroll button]"` | `NavList` | Adds an aria-label for the right scroll button |
| `id` | `NavGroup` | Used as an identifier to use when there is an `aria-labelledby` on a section element |
| `title="[text label for a nav group]"` | `NavGroup` | Adds a title shown for the group |
| `isActive` | `NavItem` |  Will add the `aria-current` attribute on a nav item when active |
| `isExpanded` | `NavExpandable` | Adds the `aria-expanded` attribute when is expanded and `hidden` attribute when collapsed |
| `title="[text label for an expandable nav]"` | `NavExpandable` | Adds a title shown for the expandable list |
| `buttonProps={[an object containing button props]}` | `NavExpandable` | Adds any additional props added to the NavExpandable `<button>`|
| `srText="[text describing an expandable nav]"` | `NavExpandable` | If defined, screen readers will read this text instead of the list title |
```